samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/SelfServiceRegistration.java

```diff
@@ -305,6 +308,35 @@ public void mary_opens_activate_page_with_token() {
         driver.get(activateURL);
     }
 
+    //Added for webauthn
+    @Then("^she sees WebAuthn factor to register$")
+    public void she_sees_webauth_factor_to_register() {
+        Assert.assertTrue(verifyPage.webAuthnRadioButton.isDisplayed());
+    }
+
+    @And("^she selects WebAuthn from the list$")
+    public void she_selects_webauth_from_the_list() {
+        verifyPage.webAuthnRadioButton.click();
+    }
+
+    @And("^she inputs the fingerprint$")
+    public void she_inputs_the_fingerprint() {
+        System.out.println("WebAuth-Start");
```

Comment:
Remove the prints. We don't need them now that tests are passing

Reply:
Thanks for the review. Have removed comments and print statements.